src/OfficeService.Models.Dto/Requests/Office/EditOfficeRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace LT.DigitalOffice.OfficeService.Models.Dto.Requests.Office
+{
+  public record EditOfficeRequest
+  {
+    public string Name { get; set; }
+    public string City { get; set; }
+    public string Address { get; set; }
+    public double Latitude { get; set; }
+    public double Longitude { get; set; }
```

Comment:
IsActive?

Reply:
added and added to request and validator